privacy-snapshot.json

```diff
@@ -86,5 +86,8 @@
   "unresponsive-rpc.url",
   "user-storage.api.cx.metamask.io",
   "verify.walletconnect.com",
-  "www.4byte.directory"
+  "www.4byte.directory",
+  "api.web3modal.org",
+  "www.4byte.directory",
```

Comment:
Looks like 4byte is added twice (?)

Reply:
Somehow made a mess when merging. I've cleared it out.